src/Microsoft.AspNetCore.SignalR.Core/HubConnectionContext.cs

```diff
@@ -282,7 +296,7 @@ private ValueTask TryWritePingAsync()
             }
             finally
             {
-                _writeLock.Release();
+                DoneWithSend();
```

Comment:
Missed renaming the call sites :). Remember that VS can do the rename for you and apply it to all the places the method is called! Just use that light bulb/Ctrl-. shortcut ;)

Reply:
aaaaaaaaahhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh